app/views/foods/_form.html.erb

```diff
@@ -1,18 +1,20 @@
-<h2>Add new food:</h2>
-<%= form_with model: @food do |form| %>
-  <p>
-    <%= form.label :name %> <br>
-    <%= form.text_field :name %>
-  </p>
+<div class="container d-flex flex-column align-items-center">
+  <h2 class="align-items-stretch">Add new food</h2>
+  <%= form_with model: @food do |form| %>
+    <p>
+      <%= form.label :name %> <br>
+      <%= form.text_field :name %>
+    </p>
```

Comment:
[OPTIONAL] For now this is very good, but it would be better if we replaced the `<p>` for `<div>` tags, bootstrap has a very good guide for form styling.
You can check it here: [Bootstrap forms](https://getbootstrap.com/docs/4.0/components/forms/)

Reply:
That was a good catch. Thanks :pray: